src/main/scala/org/dbpedia/spotlight/wikistats/ComputeStats.scala

```diff
@@ -103,7 +101,7 @@ class ComputeStats(lang: String) (implicit val sc: SparkContext,implicit val sql
     val totalSfDf = totalSfsRDD.toDF("wid", "sf2")
     val uriSfDf = wikipediaParser.getSfURI().toDF("wid", "sf1", "uri")
 
-    (totalSfDf.persist(StorageLevel.MEMORY_ONLY_SER),uriSfDf.persist(StorageLevel.MEMORY_ONLY_SER))
+    (totalSfDf.persist(StorageLevel.MEMORY_AND_DISK),uriSfDf.persist(StorageLevel.MEMORY_AND_DISK))
```

Comment:
I am creating the jsonwiki first and getting the stats from there. Persisting these intermdiate RDDs to avoid the shuffle multiple times since the same RDD is used in multiple counts